napalm/eos/eos.py

```diff
@@ -664,7 +664,7 @@ def get_re_group(res, key, default=None):
             except KeyError:
                 return default
 
-        NEIGHBOR_FILTER = "bgp neighbors vrf all | include remote AS | remote router ID |IPv[46] (Unicast|6PE):.*[0-9]+|^Local AS|Desc|BGP state"  # noqa
+        NEIGHBOR_FILTER = "bgp neighbors vrf all | include IPv[46] (Unicast|6PE):.*[0-9]+ | grep -v ' IPv[46] Unicast:/.' | remote AS |^Local AS|Desc|BGP state |remote router ID"  # noqa
```

Comment:
This regex is starting to get pretty complex.  Is it time to refactor this to TextFSM?

Reply:
> This regex is starting to get pretty complex. Is it time to refactor this to TextFSM?

Are you suggesting me to add TextFSM?

Reply:
Not really, just thinking aloud, for things to spend my copious spare time on.